_faqs/03-uninstalling-tgmpa.md

```diff
@@ -0,0 +1,54 @@
+---
+title:     How do I remove TGMPA from my theme/plugin ?
+anchor:    uninstalling-tgmpa
+permalink: /faq/uninstalling-tgmpa/
+---
+
+If your theme or plugin no longer needs **{{site.tgmpa.name}}** (TGMPA) support, then these are the steps you need to take to remove TGMPA. Some initiative may be needed depending on how the theme or plugin author has added it in, and it is assumed you are comfortable editing PHP files / FTP as needed.
+
+<ol>
+<li>Find and delete the plugin registration function. It will look something like:
+
+	{% highlight php linenos %}
+	<?php
+	/**
+	 * Required and Recommended Plugins
+	 */
+	function prefix_register_plugins() {
+
+		/**
+		 * Array of plugin arrays. Required keys are name and slug.
+		 * If the source is NOT from the .org repo, then source is also required.
+		 */
+		$plugins = array(
+
+			// Wordpress SEO
```

Comment:
@GaryJones Actually, you are: https://github.com/TGMPA/TGM-Plugin-Activation/wiki/Removing-TGM-Plugin-Activation-From-Your-Theme-Or-Plugin